pandas/tests/series/indexing/test_indexing.py

```diff
@@ -768,3 +768,24 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+def test_reset_index_drop_errmsg():
```

Comment:
I just noticed this is in the wrong file... ``reset_index`` is tested in ``pandas/tests/series/test_alter_axes.py``, not ``test_indexing.py``.

While you're at it, please replace ``# https://github.com/pandas-dev/pandas/issues/20925`` with the standard ``# GH 20925``.

Reply:
Thanks didn't know that the right file was 'test_alter_axes.py'.

Have moved the function there (now renamed as 'test_reset_index_drop_errors' based on @TomAugspurger 's comment) - also updated comment for # GH 20925